src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.Ssl.cs

```diff
@@ -209,6 +212,8 @@ public SafeBioHandle OutputBio
             }
         }
 
+        public bool HandshakeCompleted { get; set; }
```

Comment:
Instead of a property like this, consider instead having:
```C#
private bool _handshakeCompleted;
```
and
```C#
internal void MarkHandshakeCompleted() { _handshakeCompleted = true; }
```
Then instead of:
```C#
context.HandshakeCompleted = stateOk;
```
you'd do:
```C#
if (stateOk)
{
    context.MarkHandshakeCompleted();
}
```
I'm suggesting that to avoid any potential issues with HandshakeCompleted erroneously getting changed from true to false.